products.meta

```diff
@@ -150,7 +150,7 @@
     {
       "name": "chef-server",
       "type": "product",
-      "aliases": ["chef-infra-server"],
+      "aliases": ["chef-infra-server", "infra-server"],
```

Comment:
I like the aliases part. Thanks for ensuring that existing deployments will continue to work!